Umbraco-Cloud/Set-Up/Team-Members/index.md

```diff
@@ -2,24 +2,27 @@
 versionFrom: 7.0.0
 ---
 
-# Team members in the Portal and Umbraco Backoffice users
-You can add team members to your projects from the Umbraco Cloud Portal. These get automatically added as users in the backoffice of all environments for the project. The users will also be created in local clones of your development environment. The same username and password is used to login in all parts of Umbraco Cloud - Portal, backoffice and the KUDU console.
+# Team members in the Project Portal
+This article is about team members that are added via the Invite Member button in the Umbraco Cloud Portal. If you are looking for more information about Users in the Backoffice, see [Users](../../../Getting-Started/Data/Users/). Users added through the backoffice do not have access to the project portal.
+
+Team members are users that you add to your project via the Invite Member button in the Umbraco Cloud Portal. These users are automatically added as users in the backoffice of all environments for the project. These users are able to clone down the project locally and login using the same credentials they use for Cloud.
 
 ![Add team member](images/add-team-member.png)
-When adding a team member the default permission will be *Read*. If the *Admin* checkbox is ticked they will be given Admin rights to the project.
+
+When adding a team member the default permission will be *Write*. If the *Admin* checkbox is ticked they will be given Admin rights to the project.
```

Comment:
Oh did this change? 😮 You verified this, right? 🙂

Reply:
I created a new project and didn't add admin rights, and the new user was created with Write permissions.

Reply:
Yikes, wonder if the original documentation was ever correct then 😬

Awesome, will close this comment then! 🎉